src/pages/uxp/reference-js/Modules/shell/Shell.md

```diff
@@ -1,22 +1,59 @@
----
-jsDoc: true
----
 
 <a name="shell" id="shell"></a>
 
 # require('shell')
-To get an instance: `require("uxp").shell`
+To get an instance: `require("uxp").shell`<br/>
+These APIs require UXP Manifest v5 configurations. see [Launch Process](https://developer.adobe.com/photoshop/uxp/2022/guides/uxp_guide/uxp-misc/manifest-v5/#launch-process)
```

Comment:
Linking to Photoshop-specific docs might not be the best idea. I can't think of a better option right now (maybe add a manifest reference to the UXP docs and, as a second step, delete it from the product-specific sites?), but if we merge this as it is, I'm 99 % certain that it won't change in the next 5 years 😜

Reply:
Tracked in #79